src/git/objects/commits.js

```diff
@@ -47,8 +48,7 @@ export default (client, repo, store) => {
             commitMessage: message,
             commitAuthor: {
                 ...author,
-                // @TODO: Set new Date() with the good format
-                date: '2016-07-24T00:49:30+02:00',
+                date: moment().format(),
```

Comment:
You don't need `moment` for that. `(new Date()).toISOString()` does the same thing.